developer/src/kmc-kmn/src/compiler/compiler.ts

```diff
@@ -21,21 +21,24 @@ export const COMPILETARGETS_KMX =   0x01;
 export const COMPILETARGETS_JS =    0x02;
 export const COMPILETARGETS__MASK = 0x03;
 
-export interface CompilerResultMetadata {
+/**
+ * Data in CompilerResultMetadata comes from kmcmplib
```

Comment:
Is this comment still correct or should that be renamed as well?

```suggestion
 * Data in CompilerResultExtra comes from kmcmplib
```